### PR TITLE
Preparation for 1.34.0

### LINKED ISSUE #3246: Release 1.33.0
## Prerequisites

- [x] Have at least `Editor` level access to `agones-images` project.
- [x] Have permission to publish new versions of the App Engine application.
- [x] Have write access to Agones GitHub repository.
- [x] Run `git remote -v` and see:
  - [x] An `origin` remote that points to a personal fork of Agones, such as `git@github.com:yourname/agones.git`.
  - [x] An `upstream` remote that points to `git@github.com:googleforgames/agones.git`.
- [x] Have a `gcloud config configurations` configuration called `agones-images` that points to the same project.
- [x] Edit access to the [Agones Release Calendar](https://calendar.google.com/calendar/u/0?cid=Z29vZ2xlLmNvbV84MjhuOGYxOGhmYnRyczR2dTRoMXNrczIxOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
- [x] Get approval for [Release Title and Description](https://docs.google.com/document/d/1bRZCxYB8lrVcrru41b6s5D_9uU0zS49vVGdBhg0yDIY/edit)

## Steps

- [x] Run `make shell` and run `gcloud config configurations activate agones-images`.
- [x] Review merged PRs in the current milestone to ensure that they have appropriate tags.
- [x] Ensure the next stable releases in the Google Calendar have the correct version number.
- [x] `git checkout main && git pull --rebase upstream main`
- [x] Run `make pre-build-release` to ensure all example images exist on agones-images/examples repository and to deploy the 1.33.0-1 service on GCP/App Engine/Services.
- [x] Run `make sdk-update-version release_stage=before version=1.33.0` file. This command will update the version number in the sdks/install files to 1.33.0.
- [x] Create a _draft_ release with the [release template][release-template].
  - run `make release-example-image-markdown` to populate example images and append the output in `Images available with this release` section
  - [x] Draft a new release with [release draft][release-draft]. Update the `Tag version` and `Release title` with the release version and click on `Generate release notes` to generate the release notes with `Full Changelog` info for 1.33.0. Make sure to add the description. Include the `Images available with this release` section from the previous step that will be present after the `Full Changelog` and save the draft.
  - [x] copy the 1.33.0 release details from the `Full Changelog` and paste it on top of the CHANGELOG.md file
- [x] Site updated
  - [x] Create a new file named 1.33.0.md in `/site/content/en/blog/releases`. Copy the draft release content in this file (this will be what you send via email)[refer the previous release file].
  - run `make site-server` frequently to make sure everything looks fine for the release in your localhost
  - [x] In `site/content/en/docs/Installation/_index.md #agones-and-kubernetes-supported-versions`, for the current version, replace `{{% k8s-version %}}` with hardcoded Kubernetes versions supported by the current version. And add a row for the Agones release version with `{{% k8s-version %}}` as its supported Kubernetes versions.
  - [x] Run `make del-data-proofer-ignore FILENAME=1.33.0-1.md` to remove `data-proofer-ignore` attribute from previous release blog. Review all occurrences of the link_test and data-proofer-ignore attributes globally. Exclude html and release files.
  - [x] Run `make feature-shortcode-update version=1.33.0` to remove all instances of the `feature expiryVersion` shortcode, including the associated content, while preserving the rest of the content within the .md files located in site/content/en/docs. Additionally, ensure that only the block of `feature publishVersion` is removed without affecting the content.
  - [x] Add a link to previous version's documentation to nav dropdown in `site/layouts/partials/navbar.html`
  - [x] config.toml updates:
    - [x] Run `make site-config-update-version` to update the release version and sync data between dev and prod.
    - [x] Update documentation with updated example images tags.
- [x] Create PR with these changes, and merge them with an approval.
- [x] Run `git remote update && git checkout main && git reset --hard upstream/main` to ensure your code is in line
      with upstream (unless this is a hotfix, then do the same, but for the release branch)
- [x] Publish SDK packages
  - [x] Run `make sdk-shell-node` to get interactive shell to publish node package. Requires Google internal process
        to publish.
  - [x] Run `make sdk-publish-csharp` to deploy to NuGet. Requires login credentials.
        Will need [NuGet API Key](https://www.nuget.org/account/apikeys) from Agones account.
- [x] Run `make post-build-release` to build the artifacts in GCS(These files will be attached in the release notes) and to push the latest images in the release repository.
- [x] Run `make shell` and run `gcloud config configurations activate <your development project>` to switch Agones
      development tooling off of the `agones-images` project.
- [x] Do a `helm repo add agones https://agones.dev/chart/stable` / `helm repo update` and verify that the new
      version is available via the command `helm search repo agones --versions --devel`.
- [x] Do a `helm install --namespace=agones-system agones agones/agones` or `helm install --create-namespace --namespace=agones-system agones agones/agones` if the namespace was deleted and a smoke test to confirm everything is working.
- [x] Attach all assets found in the cloud storage with 1.33.0 to the draft GitHub Release.
- [x] Copy any review changes from the release blog post into the draft GitHub release.
- [x] Publish the draft GitHub Release.
- [x] Run `make release-branch` to create a release branch.
- [x] Email the [mailing list][list] with the release details (copy-paste the release blog post).
- [x] Paste the announcement blog post to the #users Slack group.
- [x] Post to the [agonesdev](https://twitter.com/agonesdev) Twitter account.
- [x] Run `git checkout main`.
- [x] Then increment the `base_version` in [`build/Makefile`][build-makefile]
- [x] Run `make sdk-update-version release_stage=after version=1.33.0` file. This command will update the version number in the sdks/install files to 1.33.0+1-dev.
- [ ] Create PR with these changes, and merge them with approval
- [ ] Close this issue. _Congratulations!_ - the release is now complete! :tada: :clap: :smile: :+1:

[list]: https://groups.google.com/forum/#!forum/agones-discuss
[release-template]: https://github.com/googleforgames/agones/blob/main/docs/governance/templates/release.md
[release-draft]: https://github.com/googleforgames/agones/releases
[build-makefile]: https://github.com/googleforgames/agones/blob/main/build/Makefile

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #3246

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/governance/templates/release_issue.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Attach all assets found in the cloud storage with {version} to the draft GitHub Release.
 - [ ] Copy any review changes from the release blog post into the draft GitHub release.
 - [ ] Publish the draft GitHub Release.
-- [ ] Run `make release-branch` to create a release branch.
-- [ ] Email the [mailing list][list] with the release details (copy-paste the release blog post).
+- [ ] Run `make release-branch` to create a release branch and run `gcloud config configurations activate <your development project>` to switch Agones development tooling off of the `agones-images` project.
+- [ ] Email the `mailing list` with the release details (copy-paste the release blog post). `Internal Agones Release Guide` has the mailing list.
```
Comment (gongmax) [inline docs/governance/templates/release_issue.md:62]: nit: Not sure whether it matters but `Internal Agones Release Guide` here is not very clear to those who don't know about the doc. Maybe explicitly call out the mailing lists here.

Comment (Kalaiselvi84) [reply docs/governance/templates/release_issue.md:62]: If my understanding is correct, we should refrain from sharing the internal document in a public forum. @markmandel, please correct me if I'm mistaken.

Comment (markmandel) [reply docs/governance/templates/release_issue.md:62]: We can link to the public google group - that's fine. Maybe something like the following:

```suggestion
- [ ] Email all mailing lists with the release details (copy-paste the release blog post). 
  - [ ] Public (agones discuss)[list] mailing list
  - [ ] Googlers: Check the [Internal Release Guide](go/agones-release-guide) for extra steps at this stage.
```

We use go links in a few projects to link to internal stuff - but also being deliberately vague 😄

Comment (Kalaiselvi84) [reply docs/governance/templates/release_issue.md]: I have included both the public email list and the Googlers email list in the [Internal] Agones Release Guide and added the corresponding link(go/agones-release-guide). Does it look good?

File: docs/governance/templates/release_issue.md [hunk 2]
```diff
@@ -58,8 +58,8 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Attach all assets found in the cloud storage with {version} to the draft GitHub Release.
 - [ ] Copy any review changes from the release blog post into the draft GitHub release.
 - [ ] Publish the draft GitHub Release.
-- [ ] Run `make release-branch` to create a release branch.
-- [ ] Email the [mailing list][list] with the release details (copy-paste the release blog post).
+- [ ] Run `make release-branch` to create a release branch and run `gcloud config configurations activate <your development project>` to switch Agones development tooling off of the `agones-images` project.
+- [ ] Email the `mailing list` with the release details (copy-paste the release blog post). Refer [Internal Mailing list posting](go/agones-release-guide).
```
Comment (markmandel) [inline docs/governance/templates/release_issue.md:62]: Minor nits on grammar:

```suggestion
- [ ] Email mailing lists with the release details (copy-paste the release blog post). Refer to the [Internal Mailing list posting guide](go/agones-release-guide) for details.
```

Other than that, LGTM.

Comment (Kalaiselvi84) [reply docs/governance/templates/release_issue.md]: Done!

**Review by markmandel — APPROVED** [review]

**Pull request merged**